### ISSUE TITLE
Still active?

### ISSUE THREAD

**Issue by raspbeguy** [solusipse/fiche#107]
Hello, I just wish to know clearly if the project is dead. Issue #100 asked the same and never got an official answer.

If @solusipse doesn't want to maintain this project anymore and some people are still wanting to contribute, could they be the new maintainers of it? We could also simply fork it, but I think this solution is not as good.

What do you think?

**Comment by jsarenik** [comment]
The original maintainer may be busy for whatever reason. Wish them health and long life. Always feel free to fork, even to link here to your fork. When they are back they can merge whatever patches got into your fork in the meantime and carry on from there, happy to see a thriving community.

**Comment by raspbeguy** [comment]
You're right. But the thing is that I wanted to check this project activity before packaging in some OS repos. So I do not wish to create a fork that will be packaged and discover few moments later that activity came back.
